spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/filters/AbstractRateLimitFilter.java

```diff
@@ -42,7 +42,7 @@
 @RequiredArgsConstructor
 public abstract class AbstractRateLimitFilter extends ZuulFilter {
 
-    private final RateLimitProperties properties;
+    protected final RateLimitProperties properties;
```

Comment:
Keep it private, add a member in the filter it self

Reply:
maybe it makes sense to move logic from `AbstractRateLimitFilter` to separate class (in separate PR)? than this change would make sense. current design with abstract class is rather misleading and leads to such oddities in child class.